config/permissions.yml

```diff
@@ -1237,6 +1237,13 @@ feature_orga_branding_edit:
     label: 'Permission to enable editing of orga branding (css styling).'
     loginRequired: true
     parent: [area_manage_orgas, area_manage_orgas_all]
+feature_orga_new_procedure_edit:
```

Comment:
If it possible I would ask to rename the permission to something (in my mind) more speakable/readable like one of these:
```
feature_manage_procedure_creation_permission
feature_administrate_procedure_creation_permission
feature_set_procedure_creation_permission
feature_modify_procedure_creation_permission
```
What do you think?

Reply:
Good idea, I will update it

Reply:
what about feature_orga_create_procedure ?